rules/react.js

```diff
@@ -36,30 +36,24 @@ module.exports = {
 		"react/self-closing-comp": 2, // Prevent extra closing tags for components without children
 		"react/sort-comp": [ 1, {
 			"order": [
-				"lifecycle",
+				"mixins",
+				"lux",
+				"props",
+				"state",
 				"everything-else",
 				"render"
 			],
 			"groups": {
-				"lifecycle": [
-					"displayName",
-					"mixins",
+				"lux": [
+					"stores",
+					"getActions"
```

Comment:
There is another one we hardly ever use, but might be worth adding `getActionGroup`